components/lcd_touch/esp_lcd_touch_ft5x06/esp_lcd_touch_ft5x06.c

```diff
@@ -161,8 +163,6 @@ esp_err_t esp_lcd_touch_new_i2c_ft5x06(const esp_lcd_panel_io_handle_t io, const
         }
```

Comment:
I think, the best solution is, assign `NULL` when failed:
```
*out_touch = NULL;
```

Reply:
I believe we should not modify the user-provided address when an error occurs. The return value allows the user to fully determine success or failure.